packages/i18nify-js/src/modules/currency/data/currencies.ts

```diff
@@ -1,98 +1,578 @@
+    conversionRate: 100
+  },
+  ETB: {
+    symbol: 'ብር',
+    name: 'Ethiopian Birr',
+    lowerUnitName:'Santim',
+    conversionRate: 100
+  },
+  EUR: {
+    symbol: '€',
+    name: 'Euro',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  FJD: {
+    symbol: 'FJ$',
+    name: 'Fijian Dollar',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  GBP: {
+    symbol: '£',
+    name: 'British Pound',
+    lowerUnitName: 'Penny',
+    conversionRate: 100
+  },
+  GHS: {
+    symbol: 'GH₵',
+    name: 'Ghanaian Cedi',
+    lowerUnitName: 'Pesewa',
+    conversionRate: 100
+  },
+  GIP: {
+    symbol: 'GIP',
+    name: 'Gibraltar Pound',
+    lowerUnitName: 'Penny',
+    conversionRate: 100
+  },
+  GMD: {
+    symbol: 'D',
+    name: 'Gambian Dalasi',
+    lowerUnitName: 'Butut',
+    conversionRate: 100
+  },
+  GTQ: {
+    symbol: 'Q',
+    name: 'Guatemalan Quetzal',
+    lowerUnitName: 'Centavo',
+    conversionRate: 100
+  },
+  GYD: {
+    symbol: 'G$',
+    name: 'Guyanese Dollar',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  HKD: {
+    symbol: 'HK$',
+    name: 'Hong Kong Dollar',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  HNL: {
+    symbol: 'HNL',
+    name: 'Honduran Lempira',
+    lowerUnitName: 'Centavo',
+    conversionRate: 100
+  },
+  HRK: {
+    symbol: 'kn',
+    name: 'Croatian Kuna',
+    lowerUnitName: 'Lipa',
+    conversionRate: 100
+  },
+  HTG: {
+    symbol: 'G',
+    name: 'Haitian Gourde',
+    lowerUnitName: 'Centime',
+    conversionRate: 100
+  },
+  HUF: {
+    symbol: 'Ft',
+    name: 'Hungarian Forint',
+    lowerUnitName: 'Fillér',
+    conversionRate: 100
+  },
+  IDR: {
+    symbol: 'Rp',
+    name: 'Indonesian Rupiah',
+    lowerUnitName: 'Sen',
+    conversionRate: 100
+  },
+  ILS: {
+    symbol: '₪',
+    name: 'Israeli New Shekel',
+    lowerUnitName: 'Agora',
+    conversionRate: 100
+  },
+  INR: {
+    symbol: '₹',
+    name: 'Indian Rupee',
+    lowerUnitName: 'Paisa',
+    conversionRate: 100
+  },
+  JMD: {
+    symbol: 'J$',
+    name: 'Jamaican Dollar',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  KES: {
+    symbol: 'Ksh',
+    name: 'Kenyan Shilling',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  KGS: {
+    symbol: 'Лв',
+    name: 'Kyrgystani Som',
+    lowerUnitName: 'Tyiyn',
+    conversionRate: 100
+  },
+  KHR: {
+    symbol: '៛',
+    name: 'Cambodian Riel',
+    lowerUnitName: 'Sen',
+    conversionRate: 100
+  },
+  KYD: {
+    symbol: 'CI$',
+    name: 'Cayman Islands Dollar',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
+  },
+  KZT: {
+    symbol: '₸',
+    name: 'Kazakhstani Tenge',
+    lowerUnitName: 'Tiyn',
+    conversionRate: 100
+  },
+  LAK: {
+    symbol: '₭',
+    name: 'Laotian Kip',
+    lowerUnitName: 'Att',
+    conversionRate: 100
+  },
+  LKR: {
+    symbol: 'රු',
+    name: 'Sri Lankan Rupee',
+    lowerUnitName: 'Cent',
+    conversionRate: 100
```

Comment:
Add this only if it is different from `100`. 
Use 100 as default fallback in case it is not present.